test/verify/check-system-realms

```diff
@@ -1160,16 +1160,20 @@ class TestPackageInstall(packagelib.PackageCase):
         # case 1: disable PackageKit
         m.execute("systemctl mask packagekit; systemctl stop packagekit.service || true")
         self.login_and_go("/system")
-        b.wait_text(self.domain_sel, "Join domain")
+        b.reload()
+        b.enter_page("/system")
```

Comment:
You don't need this -- the line before just logged into cockpit. This slows down the test in the best case, and in the worst it cases race conditions. Same thing for all other reloads that you added -- sorry for the confusing advice on IRC then, I didn't look at that actual test here.

Reply:
No worries, I assumed reloading would take us out of the system page and had kept it. But running the tests now without it still has them pass when they weren't initially.

Reply:
No, there is nothign to "slow down". wait_text() already waits for up to 15 seconds -- if it does not happen by then, it's broken (the detection does not nearly take that long)